package.json

```diff
@@ -3,11 +3,11 @@
   "private": true,
   "sideEffects": false,
   "scripts": {
-    "build": "remix build",
+    "build": "run-s build:*",
```

Comment:
This seems like something that will affect a lot of things. Our docker file uses the build command, our testing workflows use it, its used for pre-tests and so on. How will that change affect this?